src/main/resources/config/liquibase/changelog/20171121150234_changelog.xml

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
why is this changelog deleted?

Reply:
it didn't worked to build the Database, till I deleted this changlog and create the new changelog: 20171122234229_changelog.xml